### PR TITLE
Review

### PR DESCRIPTION
**Pull request by oceanlvr**

### CODE REVIEW

File: example/case.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                  console.log("success");
+                }
+              })
+              res.render("details", { data: reqData });
+            }
+          }
+        });
+      }
+    })
+});
```
Comment (oceanlvr) [inline example/case.js:58]: /review fix the callback hell problem

Comment (oceanlvr) [inline example/case.js:58]: /review fix the callback hell problem

Comment (oceanlvr) [inline example/case.js:58]: /review fix the callback hell problem

Comment (oceanlvr) [reply example/case.js:58]: /refactor refactor JavaScript code by ES2015.

Comment (oceanlvr) [reply example/case.js:58]: /refactor refactor JavaScript code by ES2015.

Comment (oceanlvr) [reply example/case.js:58]: /review fix the callback hell problem

Comment (oceanlvr) [reply example/case.js:58]: /refactor fix the callback hell problem

Comment (oceanlvr) [reply example/case.js:58]: /refactor fix the callback hell problem

Comment (oceanlvr) [reply example/case.js:58]: /refactor fix the callback hell problem

Comment (oceanlvr) [reply example/case.js:58]: /refactor fix the callback hell problem

**Comment by ZhengLiJing** [comment]
ping

File: example/case.js (hunk 1 above)
Comment (hxu296) [reply example/case.js:58]: /review does this code adhere to eslint standard?

**Comment by ikun666666666** [comment]
你好

**Comment by KillerXV** [comment]
who am i

**Comment by ikv163** [comment]
/chatgp show me python code to print hello world

**Comment by sobhanbera** [comment]
/chatgpt create a goland backend rest API endpoint to add user's to SQL database

**Comment by xixiaoxx** [comment]
介绍一下你自己

**Comment by malatya001112-lgtm** [comment]
İNCELE
İ

**Pull request closed without merging**